20FinalProject/index.html

```diff
@@ -1,10 +1,125 @@
 <!DOCTYPE html>
```

Comment:
What you did is all very much correct; I would have liked to see a little more though...

All in all, very good work this session! Excited to see your progress through the intermediate course